frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -422,234 +431,240 @@
-                    <pre>timestamp: {timestamp}</pre>
-                    <pre>expiresAt: {expiresAt}</pre>
-                    <pre>thread: {JSON.stringify(msg.thread, null, 4)}</pre>
-                {/if}
+                    {#if debug}
+                        <pre>EventIdx: {eventIndex}</pre>
+                        <pre>MsgIdx: {msg.messageIndex}</pre>
+                        <pre>MsgId: {msg.messageId}</pre>
+                        <pre>Confirmed: {confirmed}</pre>
+                        <pre>ReadByThem: {readByThem}</pre>
+                        <pre>ReadByUs: {readByMe}</pre>
+                        <pre>Pinned: {pinned}</pre>
+                        <pre>edited: {msg.edited}</pre>
+                        <pre>failed: {failed}</pre>
+                        <pre>timestamp: {timestamp}</pre>
+                        <pre>expiresAt: {expiresAt}</pre>
+                        <pre>thread: {JSON.stringify(msg.thread, null, 4)}</pre>
+                    {/if}
 
-                {#if showChatMenu}
-                    <ChatMessageMenu
-                        bind:this={messageMenu}
-                        {chatId}
-                        {isProposal}
-                        {inert}
-                        {publicGroup}
-                        {confirmed}
-                        {failed}
-                        canShare={canShare()}
-                        {me}
-                        {canPin}
-                        {canTip}
-                        {pinned}
-                        {supportsReply}
-                        {canQuoteReply}
-                        {threadRootMessage}
-                        {canStartThread}
-                        {multiUserChat}
-                        {msg}
-                        canForward={client.canForward(msg.content)}
-                        {canBlockUser}
-                        {canEdit}
-                        {canDelete}
-                        {canUndelete}
-                        {canRevealDeleted}
-                        {canRevealBlocked}
-                        {crypto}
-                        translatable={(client.getMessageText(msg.content) ?? "").length > 0}
-                        {translated}
-                        on:collapseMessage
-                        on:forward
-                        on:reply={reply}
-                        on:retrySend
-                        on:upgrade
-                        on:initiateThread
-                        on:deleteFailedMessage
-                        on:replyPrivately={replyPrivately}
-                        on:editMessage={editMessage}
-                        on:tipMessage={tipMessage}
-                        on:reportMessage={() => (showReport = true)}
-                        on:cancelReminder={cancelReminder}
-                        on:remindMe={() => (showRemindMe = true)} />
+                    {#if showChatMenu}
+                        {#if true}
```

Comment:
This can go

Reply:
Ahh is this meant to be `#if intersecting`?

Reply:
lol -yes! literally the whole point. I was just experimenting with something else. Doh.